Source/Utils/WarpXUtil.H

```diff
@@ -197,6 +197,17 @@ void getCellCoordinates (int i, int j, int k,
 int
 safeCastToInt(amrex::Real x, const std::string& real_name);
 
+/**
+* \brief Do a safe cast of a real to aa long
```

Comment:
Typo "aa long"

Reply:
Thanks for catching this, this is fixed now.